### PR TITLE
Update indentation level - activations.py

### PR DESCRIPTION
**Pull request by Frightera**
This PR changes the indentation level of the docstrings to 4 spaces.

### CODE REVIEW

File: keras/activations.py [hunk 1]
```diff
@@ -358,7 +358,8 @@ def gelu(x, approximate=False):
         if `approximate` is `False`.
 
     Reference:
-      - [Gaussian Error Linear Units (GELUs)](https://arxiv.org/abs/1606.08415)
+        - [Gaussian Error Linear Units (GELUs)](
+            https://arxiv.org/abs/1606.08415)
     """
```
Comment (haifeng-jin) [inline keras/activations.py:363]: We keep these markdowns always on the same line and add a `# noqa: E501` after `"""`.

File: keras/activations.py [hunk 2]
```diff
@@ -535,6 +536,8 @@ def serialize(activation, use_legacy_format=False):
 
     Args:
         activation : Function object.
+        use_legacy_format: Boolean, whether to use the legacy format for
+            serialization.
```
Comment (haifeng-jin) [inline keras/activations.py:540]: Add "Defaults to False." to the end.

File: keras/activations.py [hunk 3]
```diff
@@ -607,9 +610,11 @@ def deserialize(name, custom_objects=None, use_legacy_format=False):
...
 
     Args:
-      name: The name of the activation function.
-      custom_objects: Optional `{function_name: function_obj}`
-        dictionary listing user-provided activation functions.
+        name: The name of the activation function.
+        custom_objects: Optional `{function_name: function_obj}`
+            dictionary listing user-provided activation functions.
+        use_legacy_format: Boolean, whether to use the legacy format for
+            deserialization.
```
Comment (haifeng-jin) [inline keras/activations.py:617]: same here.

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Review by haifeng-jin — APPROVED** [review]

**Review by haifeng-jin — APPROVED** [review]

**Comment by Frightera** [comment]
@gbaned Any idea why copybara closed the PR without merging it?

**Comment by Frightera** [comment]
@gbaned Any idea why copybara closed the PR without merging it?

**Comment by gbaned** [comment]
> @gbaned  Any idea why copybara closed the PR without merging it?

Hi @Frightera I can see this PR is still open, can you please provide PR which is closed by copybara without merging it. Thank you!

**Comment by Frightera** [comment]
Hi @gbaned I meant #18036.

**Pull request merged**